Players/BereftSOTSPlayer.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections.Generic;
+using Microsoft.Xna.Framework;
+using SOTS.Items.Flails;
+using SOTS.Items.Permafrost;
+using SOTS.Items.SpiritStaves;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace TheBereftSouls.Players
+{
+    [ExtendsFromMod("SOTS")]
+    public class BereftSOTSPlayer : ModPlayer
+    {
+        #region Accessories
+        public bool VibrantEnch = false;
+        public bool FrigidEnch = false;
+
+        public List<Vector2> VesperaStoneCoords = new List<Vector2>(); // Used by Vespera ench
+        public static List<int> FrigidItems = new List<int>() // Used by Frigid Ench, needs to be updated with weapons from other mods; Maybe move somewhere else
+        {
+            ItemID.Snowball,
+            ItemID.SnowballCannon,
+            ItemID.SnowmanCannon,
+            ItemID.IceBlade,
+            ItemID.IceBow,
+            ItemID.IceBoomerang,
+            ItemID.IceSickle,
+            ItemID.NorthPole,
+            ItemID.StaffoftheFrostHydra,
+            ItemID.FlinxStaff,
+            ItemID.FrostStaff,
+            ItemID.Frostbrand,
+            ItemID.FrostDaggerfish,
+            ItemID.FlowerofFrost,
+            ItemID.WandofFrosting,
+            ItemID.BlizzardStaff,
+
+            ModContent.ItemType<ShatterBlade>(),
+            ModContent.ItemType<ShardStaff>(),
+            ModContent.ItemType<CryoCannon>(),
+            ModContent.ItemType<PermafrostSpiritStaff>(),
+            ModContent.ItemType<StormSpell>(),
+            ModContent.ItemType<FrigidJavelin>(),
+            ModContent.ItemType<HypericeClusterCannon>(),
+            ModContent.ItemType<IcicleImpale>(),
+            ModContent.ItemType<PBow>(),
+            ModContent.ItemType<Metalmalgamation>(),
+            ModContent.ItemType<ShardstormSpell>(),
+            ModContent.ItemType<NorthStar>(),
+            ModContent.ItemType<PBow>(),
+            ModContent.ItemType<PBow>(),
```

Comment:
I don't know why but the PBow reference can't be resolved so I'll review the code without these `ModContent.ItemType<PBow>()` references.

```txt
Error: /home/hutao/.local/share/Terraria/tModLoader/ModSources/TheBereftSouls/Players/BereftSOTSPlayer.cs(46,33): error CS0246: The type or namespace name 'PBow' could not be found (are you missing a using directive or an assembly reference?)
   at Terraria.ModLoader.Core.ModCompile.CompileMod(BuildingMod mod, Byte[]& code, Byte[]& pdb) in tModLoader\Terraria\ModLoader\Core\ModCompile.cs:line 424
   at Terraria.ModLoader.Core.ModCompile.BuildMod(BuildingMod mod, Byte[]& code, Byte[]& pdb) in tModLoader\Terraria\ModLoader\Core\ModCompile.cs:line 353
   at Terraria.ModLoader.Core.ModCompile.Build(BuildingMod mod) in tModLoader\Terraria\ModLoader\Core\ModCompile.cs:line 215
   at Terraria.ModLoader.Core.ModCompile.Build(String modFolder) in tModLoader\Terraria\ModLoader\Core\ModCompile.cs:line 189
   at Terraria.ModLoader.UI.UIBuildMod.<>c__DisplayClass5_0.<Build>b__0(ModCompile mc) in tModLoader\Terraria\ModLoader\UI\UIBuildMod.cs:line 34
   at Terraria.ModLoader.UI.UIBuildMod.BuildMod(Action`1 buildAction, Boolean reload) in tModLoader\Terraria\ModLoader\UI\UIBuildMod.cs:line 75
```